src/components/links/Link.tsx

```diff
@@ -18,15 +18,8 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-import styled from '@emotion/styled';
 import { LinkBaseStyled } from './LinkBaseStyled';
 
-export const Link = styled(LinkBaseStyled)`
-  font-weight: var(--echoes-font-weight-semi-bold);
-
-  --color: var(--echoes-color-background-accent-default);
-  --hover: var(--echoes-color-background-accent-default-hover);
-  --active: var(--echoes-color-background-accent-default-active);
-`;
+export const Link = LinkBaseStyled;
 
 Link.displayName = 'Link';
```

Comment:
this file doesn't do much now - do we still need the two?

Reply:
Yeah I wasn't sure.
I could also export it directly from `LinkBaseStyled.tsx` but it didn't seem very clean because we wouldn't have a file per variant anymore.

I could also rename `LinkBaseStyled.tsx` to `Link.tsx` and export from there, but I kind of liked that we could identify quickly what's the base style for the links...

It seems fine to me to have this almost empty file, it is pretty explicit that currently, the default link doesn't override anything from the base. But I'm ok to change it to something else if you want?